ripe/atlas/tools/commands/probes.py

```diff
@@ -166,6 +166,15 @@ def add_arguments(self):
                 "command."
             )
         )
+        self.parser.add_argument(
+            "--status",
+            type=int,
+            choices=[0, 1, 2, 3],
```

Comment:
Ooh, or better yet `--statuses never-connected,disconnected,abandoned` would be nice too.